scijava-ops-image/src/main/java/org/scijava/ops/image/deconvolve/VectorAccelerator.java

```diff
@@ -56,115 +53,102 @@
  * @param <T>
  * @implNote op names='deconvolve.accelerate', priority='0.'
  */
-public class VectorAccelerator<T extends RealType<T> & NativeType<T>> implements
-	Inplaces.Arity1<RandomAccessibleInterval<T>>
+public class VectorAccelerator<T extends RealType<T>> implements
```

Comment:
This Op is interesting now. There's a data structure, *not exposed by the module*, which appears as the input to an Op that users *could* (even though they shouldn't) call. This effectively makes the Op uncallable when JPMS is enforced (note that for now, in the scripts, it is fine). 

@ctrueden @elevans any opinions on this situation? Here are some initial options I see:
* Make this functionality into something that isn't an Op, possibly make it part of the `AcceleratorState`.
* Add a Hint where Ops can declare themselves as hidden.
* Leave it

Reply:
> There's a data structure, not exposed by the module, which appears as the input to an Op that users could (even though they shouldn't) call.

Arghhhhhhh :sweat_smile: Very good point.

Given the short time frame till the paper, my vote is "Leave it" and file an issue. Any user confusion about it is likely to be near zero—although if someone complains, we can expedite addressing it somehow.

Reply:
Thanks @gselzer for explaining the finer details of this issue. My short term vote is to "leave it" as well for the same reason that Curtis has, we just don't have time. Long term, I'm leaning towards demoting the `VectorAccelerator` from an Op to a helper class.

Reply:
Hmmm well I'm only leaning that way, so I'm not advocating we remove it right now :nerd_face:. For now we will leave this.